components/org.wso2.carbon.identity.application.authenticator.facebook/src/main/java/org/wso2/carbon/identity/application/authenticator/facebook/FacebookAuthenticator.java

```diff
@@ -794,4 +987,31 @@ protected String getCallbackUrl(Map<String, String> authenticatorProperties) {
             throw new RuntimeException("Error occurred while building URL.", e);
         }
     }
+
+    private boolean isTrustedTokenIssuer(AuthenticationContext context) {
+
+        ExternalIdPConfig externalIdPConfig = context.getExternalIdP();
+        if (externalIdPConfig == null) {
+            return false;
+        }
+
+        IdentityProvider externalIdentityProvider = externalIdPConfig.getIdentityProvider();
+        if (externalIdentityProvider == null) {
+            return false;
+        }
+
+        IdentityProviderProperty[] identityProviderProperties = externalIdentityProvider.getIdpProperties();
+        for (IdentityProviderProperty identityProviderProperty: identityProviderProperties) {
+            if (identityProviderProperty.getName().equals(IdPManagementConstants.IS_TRUSTED_TOKEN_ISSUER)) {
```

Comment:
Swap the constant with the variable to avoid possible null pointers

Reply:
Addressed by https://github.com/wso2-extensions/identity-outbound-auth-facebook/pull/54/commits/edb32edf8233e4325847f5f41ea84e89ffacc40f